server/src/main/java/com/linecorp/centraldogma/server/internal/storage/DirectoryBasedStorageManager.java

```diff
@@ -437,7 +438,8 @@ private static boolean isValidChildName(String name) {
             return false;
         }
 
-        return !name.endsWith(SUFFIX_REMOVED) && !name.endsWith(SUFFIX_PURGED);
+        return !name.endsWith(SUFFIX_REMOVED) && !name.endsWith(SUFFIX_PURGED) &&
+               !name.endsWith(GIT_EXTENSION); // Git repository whose name ends with '.git' is not allowed.
```

Comment:
Note: I understood if there are any users out there who use `.git` suffixes, the project/repositories just won't be loaded server start-up will be successful.

Reply:
That was my intention. 👍